src/atoms/Flex/Flex.web.js

```diff
@@ -1,3 +1,76 @@
-const Flex = () => null;
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Flex = styled(
+  ({
+    alignContent,
+    alignItems,
+    alignSelf,
+    display,
+    flex,
+    flexBasis,
+    flexDirection,
+    flexGrow,
+    flexShrink,
+    flexWrap,
+    justifyContent,
+    order,
+    children,
+    style,
+    ...props
+  }) =>
```

Comment:
Condition to have exact 1 child is missing here

Reply:
I think it's missing in native as well. @shubhnik Can you fix this?

Reply:
added